plugins/module_utils/net_map/networking_definition.py

```diff
@@ -583,10 +583,10 @@ def get_version_from_raw(
         ):
             return None
 
-        start_version = cls.__get_field_ip_net_version(
+        start_version = cls.get_field_ip_net_version(
```

Comment:
I'm not sure if this is a good idea but I didn't want to duplicate the code so I made the function public. Maybe I should be doing something else here though

Reply:
It's more than fine man, but, if was you, I'd move the entire method to a plain function, cause now it has zero "direct" relation to this class but with both classes that calls it.